protocol/x/ratelimit/keeper/keeper.go

```diff
@@ -62,11 +62,7 @@ func (k Keeper) ProcessWithdrawal(
 	amount *big.Int,
 ) error {
 	denomCapacity := k.GetDenomCapacity(ctx, denom)
-
-	newCapacityList := make([]dtypes.SerializableInt,
-		0,
-		len(denomCapacity.CapacityList),
-	)
+	newCapacityList := make([]dtypes.SerializableInt, len(denomCapacity.CapacityList))
```

Comment:
This was the previous bug, where we mistakenly initialized the initial length to `0`